File: Angular/TagAlize/src/app/subject-matter/subject-matter-view.model.ts
export class SubjectMatterView {

    private content: string;
    public get Content() { return this.content; }
    public set Content(value: string) { this.content = value; }

    private label: string;
    public get Label() { return this.label; }
    public set Label(value: string) { this.label = value; }

    public tags: Array<string>;
    public get Tags() { return this.tags; }
    public set Tags(value: Array<string>) { this.tags = value; }

}

File: API/ProvaApi/ProvaApi/Models/SubjectMatterViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProvaApi.Models
{
    public class SubjectMatterViewModel
    {
        public String Content { get; set; }
        public String Label { get; set; }
        public List<String> Tags { get; set; }
    }
}